families/config/sawtooth_config/processor/handler.py

```diff
@@ -318,6 +318,41 @@ def _append_setting_entry(state, key, value):
                     key, old_value, value)
 
 
+def _remove_setting_entry(state, key, value):
```

Comment:
I don't see this being called anywhere.   Did I missing something or is that for a future story?